src/Groups/matrices/FiniteFormOrthogonalGroup.jl

```diff
@@ -57,15 +56,15 @@ function _mod_p_to_a_kernel(G::Union{zzModMatrix, ZZModMatrix}, a, p)
       end
     end
     for h in gensk
-      g = deepcopy(E)
+      g = identity_matrix(R, n)
```

Comment:
Whats the problem with this one?

Reply:
In at least one of the functions, it redefined `E` to different things, and these objects are then captured in closures; it is not happy that `E` is redefined then. It seemed simplest to just replace all those `deepcopy` calls by direct calls to `identity_matrix` (IMHO makes it easier to read).